specs/HubProtocol.md

```diff
@@ -42,6 +46,18 @@ Example:
 }
 ```
 
+The `HandshakeResponse` message contains the following properties:
+
+* `error` - the optional error message if the server does not support the request protocol
+
+Example:
+
+```json
+{
+    "error": "Requested protocol 'messagepack' is not available."
+}
+```
+
```

Comment:
We'll need to get the Close frame encodings in both JSON and MsgPack in here as well. It's a bit of a pain, but we should keep this spec accurate.

Reply:
Done